wrapper/src/test/java/integration/DriverHelper.java

```diff
@@ -202,10 +202,12 @@ public static String getHostnameSql(DatabaseEngine databaseEngine) {
     }
   }
 
+  // The method should be used on connections with a target drivers ONLY !!!
   public static void setConnectTimeout(Properties props, long timeout, TimeUnit timeUnit) {
     setConnectTimeout(TestEnvironment.getCurrent().getCurrentDriver(), props, timeout, timeUnit);
   }
 
+  // The method should be used on connections with a target drivers ONLY !!!
```

Comment:
```suggestion
  // This method should be used on connections with a target driver ONLY!
```
this is a nit, apply to both comments